blockchain/types/transaction_signing.go

```diff
@@ -439,6 +439,7 @@ func (s londonSigner) SignatureValues(tx *Transaction, sig []byte) (R, S, V *big
 
 	// Check that chain ID of tx matches the signer. We also accept ID zero or nil here,
 	// because it indicates that the chain ID was not specified in the tx.
+	// NOTE: Kaia allow chain ID to be nil in this fork
```

Comment:
```suggestion
	// NOTE: Kaia doesn't allow chain ID to be nil in this fork
```

Reply:
Sorry, I've misunderstood once. The blow condition can allow chain ID to be nil